### PR TITLE
feat(console): add flutter guide

### PR DESCRIPTION
**Pull request by simeng-li**
## Summary

Add flutter guide

## Testing

image

image

## Checklist

- [ ] `.changeset`
- [ ] unit tests
- [ ] integration tests
- [ ] docs

OR

- [x] This PR is not applicable for the checklist

### CODE REVIEW

File: packages/console/src/assets/docs/guides/native-flutter/README.mdx
Comment (gao-sun) [inline packages/console/src/assets/docs/guides/native-flutter/README.mdx:153]: is this ok for display? (too many spaces)

File: packages/console/src/assets/docs/guides/native-flutter/README.mdx
Comment (gao-sun) [inline packages/console/src/assets/docs/guides/native-flutter/README.mdx:51]:
```suggestion
- Disable autobackup;
- Exclude sharedprefs FlutterSecureStorage used by the plugin.
```

File: packages/console/src/assets/docs/guides/native-flutter/README.mdx
Comment (gao-sun) [inline packages/console/src/assets/docs/guides/native-flutter/README.mdx:55]: in general, the code or non natural language tech stuff should be wrapped in the backquotes. there are few more places have the same issue in this article

```suggestion
1. To disable auto backup, go to your app manifest file and set the boolean value `android:allowBackup`:
```

**Review by gao-sun — APPROVED** [review]
approve to unblock - would you like to update the comments before merging?

**Comment by simeng-li** [comment]
Audit all commented issues and updated.

**Review by wangsijie — APPROVED** [review]

**Pull request merged**
